docs/hazmat/primitives/asymmetric/rsa.rst

```diff
@@ -485,6 +518,39 @@ Key interfaces
             instance.
 
 
+.. class:: RSAPrivateKeyWithSerialization
+
+    .. versionadded:: 0.8
+
+    Extends :class:`RSAPrivateKey`.
+
+    .. method:: private_numbers()
+
+        Create a
+        :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateNumbers`
+        object.
+
+        :returns: An
+            :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateNumbers`
+            instance.
+
+    .. method:: as_bytes(encoding, fmt, encryption_algorithm)
+
+        Serialize the key to bytes.
```

Comment:
This should probably have a bunch more words and describe the different valid choices.